### PR TITLE
add MIT license file

### LINKED ISSUE #6: License file missing
Hi,

Do you mind adding a license file for this project? Otherwise it is hard to contribute :)

Cheers!

### PR DESCRIPTION
**Pull request by bpostlethwaite**
resolves #6
@chriddyp @scjody @kiasaki

### CODE REVIEW

**Comment by kiasaki** [comment]
:+1:

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Plotly, Inc
```
Comment (alexander-daniel) [inline LICENSE]: 2016!

Comment (bpostlethwaite) [inline LICENSE]: nice catch! I copied this from plotlyjs. Looks like that file is outta sync too @etpinard @mdtusz ;)

**Comment by bpostlethwaite** [comment]
@chriddyp want to throw a dancer my way?

**Comment by chriddyp** [comment]
:dancer:

**Pull request merged**